tests/unit/commands/list/resources/test_resources_context.py

```diff
@@ -1,117 +1,302 @@
+                        {"__ApiId__": {"Ref": "ServerlessRestApi"}, "__Stage__": "*"},
+                    ]
+                },
+            },
+            "Type": "AWS::Lambda::Permission",
+        },
+        "ServerlessRestApi": {
+            "Properties": {
+                "Body": {
+                    "info": {"version": "1.0", "title": {"Ref": "AWS::StackName"}},
+                    "paths": {
+                        "/hello": {
+                            "get": {
+                                "x-amazon-apigateway-integration": {
+                                    "httpMethod": "POST",
+                                    "type": "aws_proxy",
+                                    "uri": {
+                                        "Fn::Sub": "arn:aws:apigateway:${AWS::Region}:lambda:path/2015-03-31/functions/${HelloWorldFunction.Arn}/invocations"
+                                    },
+                                },
+                                "responses": {},
+                            }
+                        }
+                    },
+                    "swagger": "2.0",
+                }
+            },
+            "Type": "AWS::ApiGateway::RestApi",
+        },
+        "ServerlessRestApiDeploymentf5716dc08b": {
+            "Properties": {
+                "Description": "RestApi deployment id: f5716dc08b0d213bd0f2dfb686579c351b09ae49",
+                "RestApiId": {"Ref": "ServerlessRestApi"},
+                "StageName": "Stage",
+            },
+            "Type": "AWS::ApiGateway::Deployment",
+        },
+        "ServerlessRestApiProdStage": {
+            "Properties": {
+                "DeploymentId": {"Ref": "ServerlessRestApiDeploymentf5716dc08b"},
+                "RestApiId": {"Ref": "ServerlessRestApi"},
+                "StageName": "Prod",
+            },
+            "Type": "AWS::ApiGateway::Stage",
+        },
+    },
+}
+
+SAM_FILE_READER_RETURN = {
+    "AWSTemplateFormatVersion": "2010-09-09",
+    "Transform": "AWS::Serverless-2016-10-31",
+    "Description": "sam-app-hello\nSample SAM Template for sam-app-hello\n",
+    "Globals": {"Function": {"Tracing": "Active", "Timeout": 3}},
+    "Resources": {
+        "HelloWorldFunction": {
+            "Type": "AWS::Serverless::Function",
+            "Properties": {
+                "CodeUri": "hello_world/",
+                "Handler": "app.lambda_handler",
+                "Architectures": ["x86_64"],
+                "Runtime": "python3.8",
+                "Events": {"HelloWorld": {"Type": "Api", "Properties": {"Path": "/hello", "Method": "get"}}},
+            },
+        }
+    },
+}
 
 
 class TestResourcesContext(TestCase):
+    @patch("samcli.commands.list.json_consumer.click.echo")
+    @patch("samcli.commands.list.json_consumer.click.get_current_context")
+    @patch("samcli.commands.list.resources.resources_context.ResourceMappingProducer.produce")
+    def test_resources_context_run_local_only_no_stack_name(
```

Comment:
I don't really understand this test. Seems that we're asserting a mocked value with an expected value? Not sure what this is trying to achieve.

Reply:
Fixed it